### PR TITLE
fix: allow FloatingCard wrapper to have layout modified

### PR DESCRIPTION
**Pull request by j10sanders**
### Overview

The wrapper of this component has a set `display: inline-block` which is unfortunate when you want the card to fill an entire width.  Adding the ability to change the display property on the wrapper with prop `containerDisplay`

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: DISC-653
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### PR Links and Envs

You will need to turn the feature flag on: https://tengu.codecademy.com/admin/test_user?prefill_id=62a3add47b25f875df069e01

Then visit a hub page with https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-2108

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.145-alpha.9c823b.0<br/>@codecademy/gamut-labs@32.0.8-alpha.9c823b.0<br/>@codecademy/gamut@48.0.4-alpha.9c823b.0<br/>@codecademy/monaco@0.6.53-alpha.9c823b.0<br/>@codecademy/styleguide@58.0.8-alpha.9c823b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62ab838c033a9c21a63a7022--gamut-preview.netlify.app>https://62ab838c033a9c21a63a7022--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62ab838c033a9c21a63a7022>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.145-alpha.c6ee29.0<br/>@codecademy/gamut-labs@32.0.8-alpha.c6ee29.0<br/>@codecademy/gamut@48.0.4-alpha.c6ee29.0<br/>@codecademy/monaco@0.6.53-alpha.c6ee29.0<br/>@codecademy/styleguide@58.0.8-alpha.c6ee29.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.145-alpha.3c99b6.0<br/>@codecademy/gamut-labs@32.0.8-alpha.3c99b6.0<br/>@codecademy/gamut@48.0.4-alpha.3c99b6.0<br/>@codecademy/monaco@0.6.53-alpha.3c99b6.0<br/>@codecademy/styleguide@58.0.8-alpha.3c99b6.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62acc256efede208b4817425--gamut-preview.netlify.app>https://62acc256efede208b4817425--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62acc256efede208b4817425>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.145-alpha.344187.0<br/>@codecademy/gamut-labs@32.0.8-alpha.344187.0<br/>@codecademy/gamut@48.0.4-alpha.344187.0<br/>@codecademy/monaco@0.6.53-alpha.344187.0<br/>@codecademy/styleguide@58.0.8-alpha.344187.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.145-alpha.2a3a2b.0<br/>@codecademy/gamut-labs@32.0.8-alpha.2a3a2b.0<br/>@codecademy/gamut@48.0.4-alpha.2a3a2b.0<br/>@codecademy/monaco@0.6.53-alpha.2a3a2b.0<br/>@codecademy/styleguide@58.0.8-alpha.2a3a2b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62acca185e99580b459ed42c--gamut-preview.netlify.app>https://62acca185e99580b459ed42c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62acca185e99580b459ed42c>Deploy Logs</a></p>

**Review by borisonr — COMMENTED** [review]
is there something like system.layout or system.padding we can add to line 7 to let us just set display directly on the component instead of adding a new prop? curious if anyone from the @Codecademy/gamut team can advise

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62b1f085e6a1806768dab019--gamut-preview.netlify.app>https://62b1f085e6a1806768dab019--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62b1f085e6a1806768dab019>Deploy Logs</a></p>

**Comment by j10sanders** [comment]
@Codecademy/gamut @dreamwasp can I get a review?  We'd like to ship the associated portal-app PR soon... thanks!

File: packages/styleguide/stories/Atoms/FloatingCard.stories.mdx [hunk 1]
```diff
@@ -83,6 +83,23 @@ export const MyComponent: React.FC = () => {
...
+
+The wrapper may need to be modified, such as for the `display` to be set to `block` instead of the default `inline-block`
+
+```tsx
+import { FloatingCard } from '@codecademy/gamut';
+import { DotDense } from '@codecademy/gamut-patterns';
+
+export const MyComponent: React.FC = () => {
+  return (
+    <FloatingCard pattern={DotDense} cardWrapperLayout={{ display: 'block' }}>
```
Comment (borisonr) [inline packages/styleguide/stories/Atoms/FloatingCard.stories.mdx:96]: i think you need to update the story to match the recent changes?

**Comment by j10sanders** [comment]
> > @Codecademy/gamut @dreamwasp can I get a review? We'd like to ship the associated portal-app PR soon... thanks!
> 
> @j10sanders code lgtm - can you link the figma? not sure what visual changes i'm checking for on the portal-side

oops, forgot to push that change!  just did that

File: packages/styleguide/stories/Atoms/FloatingCard.stories.mdx (hunk 1 above)
Comment (j10sanders) [reply packages/styleguide/stories/Atoms/FloatingCard.stories.mdx]: my bad - updated now!

**Comment by j10sanders** [comment]
The "Related Resources" section needs to fill to the end of the column.
Screen Shot 2022-06-21 at 6 07 18 PM

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.145-alpha.bb0e22.0<br/>@codecademy/gamut-labs@32.0.8-alpha.bb0e22.0<br/>@codecademy/gamut@48.0.4-alpha.bb0e22.0<br/>@codecademy/monaco@0.6.53-alpha.bb0e22.0<br/>@codecademy/styleguide@58.0.8-alpha.bb0e22.0<br/></pre>

**Review by dreamwasp — APPROVED** [review]
Looks great!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.146-alpha.ecffbe.0<br/>@codecademy/gamut-labs@32.1.1-alpha.ecffbe.0<br/>@codecademy/gamut@48.0.4-alpha.ecffbe.0<br/>@codecademy/monaco@0.6.53-alpha.ecffbe.0<br/>@codecademy/styleguide@58.0.9-alpha.ecffbe.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62b311006e49e709beb18c62--gamut-preview.netlify.app>https://62b311006e49e709beb18c62--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62b311006e49e709beb18c62>Deploy Logs</a></p>

**Pull request merged**